Cargo.toml

```diff
@@ -16,5 +18,4 @@ path = "src/bin.rs"
 [dependencies]
 md5 = "0"
 clap = "2"
-
-[dev-dependencies]
+helium-crypto = { git = "https://github.com/helium/helium-crypto-rs", tag = "v0.3.4", optional = true }
```

Comment:
only downside here is that now you've pinned crypto-rs to a single version which means we'll be chasing this forward

Reply:
can we start releasing it via cargo? i don't think it's up there which makes upgrading using semver logic easier

before, i was pinned to master/main which i don't love

Reply:
unfortunately we can't because the Dalek repo is not moving forward.. So not all deps are on crates.io today

Reply:
I guess we could do what the Solana guys did and push a temporary crate there with our changes up to crates.io to get this done?

Reply:
oh, that seems like a good idea to me. It would generally be nice to have helium-crypto on cargo, otherwise we're going to have a whole downstream blockage of publishing things to cargo.